src/Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn/Visitors/ReferenceItemVisitor.cs

```diff
@@ -383,6 +477,99 @@ public override void VisitPointerType(IPointerTypeSymbol symbol)
             });
         }
 
+        public override void VisitFunctionPointerType(IFunctionPointerTypeSymbol symbol)
```

Comment:
Function pointers, much like regular pointers, are illegal in VB. But it seems there is existing logic to generate "parts" for pointers in VB: <https://github.com/dotnet/docfx/pull/6805/files#diff-f05a1f9271b4d7874485ce55d806b5ef784db97e7544cd7f4353ad7c263df0f0L375>

Is this by design?

Reply:
I'm not aware of any specific reason to _intentionally support pointers for VB_. Might be the code for pointer is small and is accidentally duplicated here, but this function pointer is a big chunk of code and if we believe it is illegal for VB, leaving it as not implemented or marking it as not supported looks more clear to me.

Reply:
Thanks for the clarification, I think this should be "good" then with how I've set it up.

If there is something explicit to be done to mark it as "illegal" or to "gracefully fail" please let me know :)